src/v2/Components/Payment/SavedCreditCards.tsx

```diff
@@ -37,11 +37,11 @@ export class CreditCard extends React.Component<
   render() {
     return (
       <>
-        <BorderBox flexDirection="column" p={2} mb={2}>
+        <BorderBox flexDirection="column" p={2} mb={4}>
           <Flex justifyContent="space-between" alignItems="center">
             {/* @ts-expect-error STRICT_NULL_CHECK */}
             <CreditCardDetails {...this.props.creditCard} />
-            <Sans size="2" color="purple100">
+            <Text size="2" color="red100">
```

Comment:
We should be using `variants` (no more `size`)

Reply:
I'm gonna make an issue to just unset this prop since it probably causes more confusion than it's worth (it's a shorthand for setting width/height so it's technically valid)